src/schema/search.ts

```diff
@@ -224,10 +259,27 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
     },
     searchPosts: async (
       source,
-      args: FeedArgs & { query: string },
+      args: FeedArgs & { query: string; version: number },
       ctx,
       info,
     ): Promise<ConnectionRelay<GQLPost> & { query: string }> => {
+      if (args.version === 2) {
+        const meilieSearchRes = await searchMeili(
+          `q=${args.query}&attributesToRetrieve=post_id`,
+        );
+
+        const meilieArgs: FeedArgs & { ids: string[] } = {
+          ...args,
+          ids: meilieSearchRes.map((x) => x.post_id),
+        };
```

Comment:
What fields are available in our documents? Also, we should also have dev only API key and index so we can connect to it and use in development locally.

Reply:
Not many think top of head
- time
- post_id
- title
- content 

In this query we only need id as we do internal feed resolver anyway 🙏